server/game/cards/04.5-AaN/MatsuMitsuko.js

```diff
@@ -1,10 +1,19 @@
 const DrawCard = require('../../drawcard.js');
 
 class MatsuMitsuko extends DrawCard {
-    setupCardAbilities(ability) { // eslint-disable-line no-unused-vars
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Move a character to the conflict',
+            condition: context => this.game.isDuringConflict('military') && context.player.opponent.honor < context.player.honor,
```

Comment:
You should add `&& context.player ` after the `isDuringConflict` call to stop the game crashing in single player